devicetypes/smartthings/zigbee-thermostat.src/zigbee-thermostat.groovy

```diff
@@ -310,29 +322,40 @@ def configure() {
 	def startValues = zigbee.writeAttribute(THERMOSTAT_CLUSTER, HEATING_SETPOINT, DataType.INT16, 0x07D0) +
 			zigbee.writeAttribute(THERMOSTAT_CLUSTER, COOLING_SETPOINT, DataType.INT16, 0x0A28)
```

Comment:
you shouldn't be sending cooling setpoints to a thermostat that doesn't support them.

@PKacprowiczS had something specific in mind when he wrote this for the Lux, I'm not sure you want to send them to other devices

Reply:
I've missed that part of the code. It's fixed now.